README.md

```diff
@@ -1,354 +1,50 @@
-<div id="top"></div>
 <div align="center">
 
-<img  src="src/assets/app-logo.png"  alt="Dummygram"  width="250"  height="200">
-
-[See Live!](https://narayan954.github.io/dummygram/)
-
-### _Instagram_, dummified
-
-<div align="center">
-      <img src="https://forthebadge.com/images/badges/built-with-love.svg" />
-      <img src="https://forthebadge.com/images/badges/uses-brains.svg" />
-      <img src="https://forthebadge.com/images/badges/powered-by-responsibility.svg" />
-      <br>
-      <img src="https://img.shields.io/github/repo-size/narayan954/dummygram?style=for-the-badge" />
-      <img src="https://img.shields.io/github/issues/narayan954/dummygram?style=for-the-badge" />
-      <img src="https://img.shields.io/github/issues-raw/narayan954/dummygram?style=for-the-badge" />
-      <img src="https://img.shields.io/github/forks/narayan954/dummygram?style=for-the-badge" />
-      <img src="https://img.shields.io/github/issues-pr-closed/narayan954/dummygram?style=for-the-badge" />
-      <br>
-      <img src="https://img.shields.io/github/stars/narayan954/dummygram?style=for-the-badge" />
-      <img src="https://img.shields.io/github/contributors/narayan954/dummygram?style=for-the-badge" />
-      <img src="https://img.shields.io/github/issues-pr-closed/narayan954/dummygram?style=for-the-badge" />
-      <img src="https://img.shields.io/github/last-commit/narayan954/dummygram?style=for-the-badge" />
- </div>
-
----
+## DummyGram
 
+[See Demo Here](https://narayan954.github.io/dummygram/)
 </div>
+<img src="https://img.shields.io/github/repo-size/narayan954/dummygram?style=for-the-badge" />
```

Comment:
why did you remove rest of the badges? I think they'd still look good

Reply:
I think rest logos are irrelevant to show, bcz users don't have concern regarding no. of issues/PR's that are open or closed.

Reply:
Still, let them stay.